bitnami/prometheus-operator/templates/prometheus-operator/deployment.yaml

```
@@ -74,8 +85,8 @@ spec:
             {{- end }}
             - --logtostderr=true
             - --localhost=127.0.0.1
-            - --config-reloader-image={{ template "prometheus-operator.configmapReload.image" . }}
-            - --prometheus-config-reloader={{ template "prometheus-operator.prometheusConfigReloader.image" . }}
+            - --config-reloader-image=$(CONFIG_RELOADER_IMAGE)
+            - --prometheus-config-reloader=$(PROMETHEUS_CONFIG_RELOADER)
```

If these are environment variables, shouldn't they be specified like this?

```suggestion
            - --config-reloader-image=${CONFIG_RELOADER_IMAGE}
            - --prometheus-config-reloader=${PROMETHEUS_CONFIG_RELOADER}
```

---

No, this is a Kubernetes specific syntax for environment variables in parameters.  https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#use-environment-variables-to-define-arguments